.github/labeler.yml

```diff
@@ -0,0 +1,66 @@
+# Add 'awaiting-review' to any PR opened
+awaiting-review:
+- '**/*'
+
+'product: CLion':
+- clwb/**/*
+- cpp/**/*
+- base/**/*
+- examples/cpp/simple_project/**/*
+- intellij_platform_sdk/BUILD.clion213
+- intellij_platform_sdk/BUILD.clion221
+- intellij_platform_sdk/BUILD.clion222
+- intellij_platform_sdk/BUILD.clion223
+- intellij_platform_sdk/BUILD.clion231
+- intellij_platform_sdk/BUILD.clion232
+
+'product: Android Studio':
+- aswb/**/*
+- sdkcompat/**/*
+- examples/kotlin/**/*
+- base/**/*
+- intellij_platform_sdk/BUILD.android_studio213
+- intellij_platform_sdk/BUILD.android_studio221
+- intellij_platform_sdk/BUILD.android_studio222
+- intellij_platform_sdk/BUILD.android_studio223
+- intellij_platform_sdk/BUILD.android_studio231
+- intellij_platform_sdk/BUILD.android_studiodev
+
+'product: IntelliJ':
+- aspect/**/*
```

Comment:
can we remove `aspect/**/*`, `build_defs/**/*`, `shared/**/*`, `querysync/**/*` from this list?

Reply:
Could you please guide which label this is categorised Or should we leave it!

Reply:
just remove it please and remove third_party ones as well.

Reply:
Sure.